pkgs/by-name/bi/binary/package.nix

```diff
@@ -0,0 +1,54 @@
+{
+  lib,
+  fetchFromGitHub,
+  python3Packages,
+  appstream,
+  desktop-file-utils,
+  glib,
+  gobject-introspection,
+  libadwaita,
+  meson,
+  ninja,
+  pkg-config,
+  wrapGAppsHook4,
+}:
+python3Packages.buildPythonApplication rec {
+  pname = "binary";
+  version = "0.3";
+  pyproject = false;
```

Comment:
```suggestion
  format = "other";
```

Reply:
i've applied the change for now, but i'm a bit confused by this suggestion with https://github.com/NixOS/nixpkgs/issues/253154. is this (eventual) deprecation not meant to affect packages that use a non python-based build system like in this case? cc @figsoda

Reply:
Nowadays we use `pyproject = false`.

Why did you suggest the opposite, @SuperSandro2000 ?